bank/src/main/java/org/example/bank/Bank.java

```diff
@@ -0,0 +1,181 @@
+package org.example.bank;
+import org.example.account.Account;
+import org.example.account.AccountManager;
+import org.example.account.AccountView;
+import org.example.user.User;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Bank  {
+    protected static Scanner sc = new Scanner(System.in);
+
+    public static ArrayList<Account> accountList = new ArrayList<Account>();
+    static ArrayList<User> users = new ArrayList<>();
+    public int accountcnt =0;
+    public String nameofBank = null;
+    public static int howmanyBank = 0;
+    public int accountLog = -1;
+    private BankController controller;
+
+
+    public  Bank(){
+        this.controller = new BankController(this);
+        howmanyBank++;
+    }
+
+    public Bank(String nameofBank){
+        this();
+        this.nameofBank = nameofBank;
+    }
+
+    public void createUser() {
+        System.out.println("------------------------------------");
+        System.out.println("가입 서비스입니다. 원하시는 아이디를 입력하세요");
+        String id = sc.nextLine();
+        if(validateDuplicatedId(id)){
+            System.out.println("중복된 아이디입니다.");
+            return;
+        }
```

Comment:
```suggestion
       validateDuplicatedId(id);
```
위 메소드를 하나의 메소드에 포함시켜도 괜찮습니다.